test/extended/testdata/builds/pullsecret/pullsecret-nodejs-bc.yaml

```diff
@@ -0,0 +1,16 @@
+kind: BuildConfig
+apiVersion: v1
+metadata:
+  name: pullsecret-nodejs
+spec:
+  source:
+    git:
+      uri: "https://github.com/sclorg/nodejs-ex.git"
+  strategy:
+    type: Source
+    sourceStrategy:
+      from:
+        kind: DockerImage
+        name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
+    pullSecret:
```

Comment:
@adambkaplan looks like the indentation on you yaml is resulting in this pull secret not getting pulled in

See that it is missing when I use it as is to create the BC:  
```
gmontero ~/QE_bzs/img-src-auth $ cat pullsecret-nodejs.yaml 
kind: BuildConfig
apiVersion: v1
metadata:
  name: pullsecret-nodejs
spec:
  source:
    git:
      uri: "https://github.com/sclorg/nodejs-ex.git"
  strategy:
    type: Source
    sourceStrategy:
      from:
        kind: DockerImage
        name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
    pullSecret:
      name: local-ps
gmontero ~/QE_bzs/img-src-auth $ oc get bc -o yaml
apiVersion: v1
items:
- apiVersion: build.openshift.io/v1
  kind: BuildConfig
  metadata:
    creationTimestamp: "2019-11-15T18:35:13Z"
    name: pullsecret-nodejs
    namespace: ggmtest
    resourceVersion: "49618"
    selfLink: /apis/build.openshift.io/v1/namespaces/ggmtest/buildconfigs/pullsecret-nodejs
    uid: e4c6808a-da49-4a11-b9e1-ca736e533014
  spec:
    failedBuildsHistoryLimit: 5
    nodeSelector: null
    output: {}
    postCommit: {}
    resources: {}
    runPolicy: Serial
    source:
      git:
        uri: https://github.com/sclorg/nodejs-ex.git
      type: Git
    strategy:
      sourceStrategy:
        from:
          kind: DockerImage
          name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
      type: Source
    successfulBuildsHistoryLimit: 5
    triggers: []
  status:
    lastVersion: 1
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
gmontero ~/QE_bzs/img-src-auth $
```
when I indent `pullSecret` in by 2 spaces, it is incorporated:

```
gmontero ~/QE_bzs/img-src-auth $ oc get bc -o yaml
apiVersion: v1
items:
- apiVersion: build.openshift.io/v1
  kind: BuildConfig
  metadata:
    creationTimestamp: "2019-11-15T18:45:03Z"
    name: pullsecret-nodejs
    namespace: ggmtest
    resourceVersion: "52110"
    selfLink: /apis/build.openshift.io/v1/namespaces/ggmtest/buildconfigs/pullsecret-nodejs
    uid: e56ff415-94da-4da0-b20b-a53f2e7c49dd
  spec:
    failedBuildsHistoryLimit: 5
    nodeSelector: null
    output: {}
    postCommit: {}
    resources: {}
    runPolicy: Serial
    source:
      git:
        uri: https://github.com/sclorg/nodejs-ex.git
      type: Git
    strategy:
      sourceStrategy:
        from:
          kind: DockerImage
          name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
        pullSecret:
          name: local-ps
      type: Source
    successfulBuildsHistoryLimit: 5
    triggers: []
  status:
    lastVersion: 0
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
gmontero ~/QE_bzs/img-src-auth $ 
```

and the build ultimatey works with my changes to openshift/builder

please update this and we'll try the e2e in this PR again


```

Reply:
the corrected pullsecret-nodejs yaml:

```
gmontero ~/QE_bzs/img-src-auth $ cat pullsecret-nodejs.yaml
kind: BuildConfig
apiVersion: v1
metadata:
  name: pullsecret-nodejs
spec:
  source:
    git:
      uri: "https://github.com/sclorg/nodejs-ex.git"
  strategy:
    type: Source
    sourceStrategy:
      from:
        kind: DockerImage
        name: registry.redhat.io/rhscl/nodejs-10-rhel7:latest
      pullSecret:
        name: local-ps
gmontero ~/QE_bzs/img-src-auth $ 
```

Reply:
@gabemontero gotta love YAML. Fixed in the next push.

Reply:
yes you do :-)